module/x/peggy/keeper/keeper.go

```diff
@@ -321,65 +380,12 @@ func (k Keeper) SetOrchestratorValidator(ctx sdk.Context, val sdk.ValAddress, or
 // GetOrchestratorValidator returns the validator key associated with an orchestrator key
 func (k Keeper) GetOrchestratorValidator(ctx sdk.Context, orch sdk.AccAddress) sdk.ValAddress {
 	store := ctx.KVStore(k.storeKey)
-	return sdk.ValAddress(store.Get(types.GetOrchestratorAddressKey(orch)))
-}
-
-/////////////////////////////
-//       PARAMETERS        //
-/////////////////////////////
-
-// GetParams returns the parameters from the store
-func (k Keeper) GetParams(ctx sdk.Context) (params types.Params) {
-	k.paramSpace.GetParamSet(ctx, &params)
-	return
-}
-
-// SetParams sets the parameters in the store
-func (k Keeper) SetParams(ctx sdk.Context, ps types.Params) {
-	k.paramSpace.SetParamSet(ctx, &ps)
-}
-
-// GetBridgeContractAddress returns the bridge contract address on ETH
-func (k Keeper) GetBridgeContractAddress(ctx sdk.Context) string {
-	var a string
-	k.paramSpace.Get(ctx, types.ParamsStoreKeyBridgeContractAddress, &a)
-	return a
-}
-
-// GetBridgeChainID returns the chain id of the ETH chain we are running against
-func (k Keeper) GetBridgeChainID(ctx sdk.Context) uint64 {
-	var a uint64
-	k.paramSpace.Get(ctx, types.ParamsStoreKeyBridgeContractChainID, &a)
-	return a
-}
-
-// GetPeggyID returns the PeggyID (???)
-func (k Keeper) GetPeggyID(ctx sdk.Context) string {
-	var a string
-	k.paramSpace.Get(ctx, types.ParamsStoreKeyPeggyID, &a)
-	return a
-}
-
-// Set PeggyID sets the PeggyID (couldn't we reuse the ChainID here?)
-func (k Keeper) setPeggyID(ctx sdk.Context, v string) {
-	k.paramSpace.Set(ctx, types.ParamsStoreKeyPeggyID, v)
-}
-
-// GetStartThreshold returns the start threshold for the peggy validator set
-func (k Keeper) GetStartThreshold(ctx sdk.Context) uint64 {
-	var a uint64
-	k.paramSpace.Get(ctx, types.ParamsStoreKeyStartThreshold, &a)
-	return a
-}
-
-// setStartThreshold sets the start threshold
-func (k Keeper) setStartThreshold(ctx sdk.Context, v uint64) {
-	k.paramSpace.Set(ctx, types.ParamsStoreKeyStartThreshold, v)
-}
+	bz := store.Get(types.GetOrchestratorAddressKey(orch))
+	if bz == nil {
+		return nil
+	}
 
-// logger returns a module-specific logger.
```

Comment:
One thing ive done in the Oracle is just to use the `sdk.AccAddress`. Another address based refactor we could look into with questionable benefits.